### PR TITLE
chore: improve changelogs format

### PR DESCRIPTION
**Pull request by lumtis**
# Description

Make some updates in the changelog format

- Separate changes by PR type
- Simplify PR URL

### CODE REVIEW

**Review by ws4charlie — APPROVED** [review]
This looks really nice and clean.

File: changelog.md [hunk 1]
```diff
@@ -1,45 +1,76 @@
...
+
+### Fixes
+
+* [1195](https://github.com/zeta-chain/node/pull/1195) - added upgrade name, and allow download. allows to test release
+* [1153](https://github.com/zeta-chain/node/pull/1153) - address `cosmos-gosec` lint issues
+* [1128](https://github.com/zeta-chain/node/pull/1228) - adding namespaces back in rpc
+* [1245](https://github.com/zeta-chain/node/pull/1245) - set unique index for generate cctx
+* [1250](https://github.com/zeta-chain/node/pull/1250) - remove error return in `IsAuthorized`
+* [1261](https://github.com/zeta-chain/node/pull/1261) - Ethereum comparaison checksum/non-checksum format
+* [1264](https://github.com/zeta-chain/node/pull/1264) - Blame index update by @kevinssgh
```
Comment (kingpinXD) [inline changelog.md:35]: We can decide fo or against keeping github handles , but lets keep it uniform ? 
In my opinion the names are redundant as the prs are already linked

Comment (lumtis) [reply changelog.md:35]: Yes, I think it's clearer removing handles, this is a typo

Comment (lumtis) [reply changelog.md]: [b731d6c](https://github.com/zeta-chain/node/pull/1390/commits/b731d6cb9deff879041346cd66fbf5c7b37bd95c)

**Review by kingpinXD — APPROVED** [review]

**Pull request merged**